### PR TITLE
Replaces Overcrowded Detective Locker With Vendor

### PR DESCRIPTION
**Pull request by Surrealaser**
:cl:
tweak: Overstuffed detective lock replaced with vendor.
/:cl:

### CODE REVIEW

File: code/FulpstationCode/sec_expanded_loadouts/detective_vendor.dm [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+					/obj/item/coin/antagtoken = 1,
+					/obj/item/storage/fancy/cigarettes/cigpack_robustgold = 4,
+					/obj/item/storage/box/gum/nicotine = 2,
+					/obj/item/lighter = 4,
+					/obj/item/clothing/mask/cigarette/pipe = 4,
+					/obj/item/storage/fancy/cigarettes/cigars/havana = 12,
+					/obj/item/storage/fancy/cigarettes/cigars/cohiba = 12)
+
+	refill_canister = /obj/item/vending_refill/detective
+	default_price = 650
```
Comment (ATH1909) [inline code/FulpstationCode/sec_expanded_loadouts/detective_vendor.dm]: damn, son, ain't this a bit too expensive?

Comment (Surrealaser) [reply code/FulpstationCode/sec_expanded_loadouts/detective_vendor.dm]: That's copy pasta; I don't mind changing it, not a big deal.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -218,7 +218,8 @@
 	new /obj/item/pinpointer/crew(src)
 	new /obj/item/clothing/shoes/laceup/digitigrade(src) //fulp
 	new /obj/item/binoculars(src)
```
Comment (ATH1909) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:220]: hey, you never added these binoculars to the vending machine...

also maybe consider adding some handcuffs to the vending machine as well, but they're optional, I guess

Comment (ATH1909) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:220]: you also never added the digitigrade laceup shoes to the vending machine

Comment (ATH1909) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:220]: also also, you never added a way to buy an /obj/item/vending_refill/detective from cargo

**Comment by GuillaumePrata** [comment]
So a vendor spawns on the room, the wardrobe is replaced with a vendor or just the clothes are removed and the parts added in to build the vendor?
Tried to read the code directly but didnt got a answer.

**Comment by Surrealaser** [comment]
@GuillaumePrata Basically, closet spawns, vendor spawns at the same location, closet is deleted.

**Comment by remove32** [comment]
the best way to make a map change without changing the map

File: code/FulpstationCode/sec_expanded_loadouts/detective_goggles.dm [hunk 3]
```diff
@@ -14,8 +14,8 @@
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
 
 /obj/item/clothing/glasses/detective
-	name = "detective glasses"
-	desc = "Stylish glasses with integrated medical, diagnostic and security HUDs and reagent scanning used by detectives. The Meson Scanner mode lets you see basic structural and terrain layouts through walls. Flash proofing compromised to accomodate HUD integration."
+	name = "detective's glasses"
+	desc = "Stylish glasses with integrated medical, diagnostic and security HUDs and reagent scanning used by detectives. The Meson Scanner mode lets you see basic structural and terrain layouts through walls. WARNING! Flash proofing has been compromised to accomodate HUD integration."
```
Comment (ATH1909) [inline code/FulpstationCode/sec_expanded_loadouts/detective_goggles.dm:18]: is it "flash proofing" or "flash-proofing"? I forget (and genuinely don't know)

Comment (Surrealaser) [reply code/FulpstationCode/sec_expanded_loadouts/detective_goggles.dm:18]: I don't think it matters.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 2 above)
Comment (Surrealaser) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:220]: This has been addressed.

**Comment by TheSwain** [comment]
MERGED. Let's see how this does.

**Pull request merged**